README.md

```diff
@@ -66,9 +63,10 @@ Refer to [AutoScaler UAT guide](src/acceptance/README.md) to run acceptance test
 
 ## Use service
 
-To use the service to auto-scale your application, firstly create the service, and then bind to you application with policy as parameter. 
+To use the service to auto-scale your application, firstly enable service access to all or specific orgs, then create the service, and then bind to your application with policy as parameter.
```

Comment:
we need to make it clear that we need admin user to enable service access, then service creation/bind can be a normal user.